controllers/past-grants/views/recipient-detail.njk

```diff
@@ -0,0 +1,116 @@
+
+{% set bodyClass = 'has-static-header' %}
+
+{% block content %}
+    <main role="main" id="content">
+        <div class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+            {{ breadcrumbTrail(breadcrumbs) }}
+
+            <div class="content-sidebar">
+                <div class="content-sidebar__primary">
+                    <div class="u-constrained-wide accent--pink">
+                        <h1>{{ title }}</h1>
+
+                        <dl class="o-definition-list o-definition-list--compact project-details">
+                            <dt>Type</dt>
+                            <dd>
+                                {{ buildOrgTypeList(organisation) }}
+                            </dd>
+
+                            {% if organisation.charityNumber %}
+                                <dt>Charity number</dt>
+                                <dd>{{ organisation.charityNumber }}</dd>
+                            {% endif %}
+
+                            {% if organisation.companyNumber %}
+                                <dt>Company number</dt>
+                                <dd>{{ organisation.companyNumber }}</dd>
+                            {% endif %}
+
+                            <dt>Location</dt>
+                            <dd>
+                                {% if recipientLocalAuthorities.length === 1 %}
+                                    {% set grant = recipientGrants[0] %}
+                                    {{ buildLocationList(grant.beneficiaryLocation) }}
+                                {% else %}
+                                    Multiple locations
+                                {% endif %}
+                            </dd>
+
+                        </dl>
+
+                        <p class="u-margin-top-l">
+                            {% set orgNameSafe = organisation.name | urlencode %}
+                            <strong>
+                                <a href="https://www.google.co.uk/search?q={{ orgNameSafe }}">
+                                    Search for {{ organisation.name }} with Google
+                                </a>
+                            </strong>
+                        </p>
+
+                    </div>
+                </div>
+
+                <div class="content-sidebar__secondary u-tone-background-tint u-padded">
+                    <h2>Awarding programmes</h2>
+                        <ul>
+                            {% for programme in recipientProgrammes %}
+                                <li class="u-margin-bottom t5">
+                                    <a class="accent--pink a--text"
+                                       href="{{ pgLink('programme', programme.label) }}">
+                                        {{ programme.label }}</a>
+                                    {% if recipientProgrammes.length > 1 %}
+                                        <br />
+                                        <small>
```

Comment:
Not sure about this award count? It is not going to say "1" a lot?

Reply:
Possibly – it'll only appear for orgs who've had grants in multiple programmes, though.

Reply:
Oh yeah, was mostly going on the original screenshot tbh and didn't spot that.